pynuodb/crypt.py

```diff
@@ -261,46 +262,20 @@ def computeSessionKey(self, clientKey, verifier):
 class RC4Cipher(object):
 
     def __init__(self, key):
+        self.cipher = Cipher(algorithms.ARC4(key), mode=None, backend=default_backend()).encryptor()
+
+    def transform(self, data):
+        # Cipher expects bytes
+        if systemVersion == '3' and type(data) == str:
+            data = bytes(data, "latin-1")
+        transformed = self.cipher.update(data)
```

Comment:
What does 'systemVersion' mean? Python version?

Reply:
Yeah. I'm not sure why this file uses systemVersion, but I was just keeping with the style found in this file.

Reply:
Should there be an error message or exception thrown if the systemVersion != '3' or type(data) != str?

Reply:
Nope. We support python 2. And its not incorrect to supply bytes to transform (as long as those bytes are from a "latin-1" encoding..not sure why we don't use utf-8 but that's an issue for another day)

Reply:
I was just about to wonder why we don't use utf8.  Sigh.